include/gatekeeper_net.h

```diff
@@ -40,4 +42,14 @@ int gatekeeper_init_network(struct net_config *net_conf);
 
 void gatekeeper_free_network(void);
 
+/*
+ * Each functional block listens on one or more lcores. The lcore
+ * identifier also serves as a queue identifier.
+ */
+static inline uint16_t
+get_queue_id(void)
+{
+	return (uint16_t)rte_lcore_id();
```

Comment:
The lcore IDs indeed do *not* always start from 0. They are used exactly as they are identified with any command line parameters.

Therefore, I've added the array you suggested, which allows us to use queue numbers starting from 0 with any set of lcore IDs.